screen_brightness_control/__init__.py

```diff
@@ -466,12 +466,13 @@ def get_identifier(self, monitor: dict = None) -> Tuple[str, Union[int, str]]:
             if value is not None:
                 return key, value
 
-    def set_brightness(self, value: int, no_return: bool = True, force: bool = False) -> Union[None, int]:
+    def set_brightness(self, value: str, no_return: bool = True, force: bool = False) -> Union[None, int]:
```

Comment:
Could this type hint be changed to `Union[int, str]`, since integer brightness values are definitely supported?
And also in the below docstring would be great